VATSpy.dat

```diff
@@ -17965,8 +17965,9 @@ EGYP|Mount Pleasant (Monte Agradable)|ISLAND|EGYP
 EHAA|Amsterdam||EHAA
 EHMC|Dutch Mil (Up to FL245, including N and E)||EHMC
 EHML|Bandbox CRC (Military)||EHAA
-EIDW|Dublin ACC (Up to FL245) - Shannon||EIDW
-EISN|Shannon||EISN
+EIDW|Dublin ACC (Up to FL245) - Shannon|EIDW|EIDW
+EISN|Shannon ACC - Shannon|EISN|EISN

```

Comment:
Not nessecerily to add alias on these two since they match the FIR.

Reply:
Fixed